src/lasp_default_broadcast_distribution_backend.erl

```diff
@@ -502,12 +506,51 @@ init([]) ->
                 counter=Counter,
                 sync_counter=0,
                 store=Store,
+                to_broadcast=ToBroadcast,
                 gc_counter=GCCounter}}.
 
 %% @private
 -spec handle_call(term(), {pid(), term()}, #state{}) ->
     {reply, term(), #state{}}.
 
+handle_call(perform_broadcast,
+            _From,
+            #state{to_broadcast=ToBroadcast0}=State) ->
+
+    dict:fold(fun(Id, {Id, Type, Metadata, Value} = Payload, ok) ->
+                    PeerCount = length(plumtree_broadcast:broadcast_members()),
+                    log_transmission({broadcast, Payload}, PeerCount),
+                    Clock = orddict:fetch(clock, Metadata),
+                    Broadcast = #broadcast{id=Id,
+                                           clock=Clock,
+                                           type=Type,
+                                           metadata=Metadata,
+                                           value=Value},
+                    {Time, _Result} = timer:tc(plumtree_broadcast,
+                                              broadcast,
+                                              [Broadcast, ?MODULE]),
+                    lasp_logger:extended("Time to broadcast: ~p", [Time]),
+                    ok
+              end, ok, ToBroadcast0),
+
+    %% Buffer latest update for that state.
```

Comment:
I think this comment is not correct. This is the point we flush the broadcast buffer.